.gitlab-ci.yml

```diff
@@ -46,6 +46,7 @@ snapshot:
 release:
   stage: release
   artifacts:
+    when: always
```

Comment:
This line will save us in case the gitlab build fails because of random crap again